pkg/providers/instancetype/suite_test.go

```diff
@@ -425,6 +430,162 @@ var _ = Describe("InstanceTypes", func() {
 			Expect(spotPrice).To(BeNumerically("<", cheapestODPrice))
 		}
 	})
+	It("should consider the minValues from any requirement for capping InstanceTypeOptions", func() {
```

Comment:
I don't think that we need to cap anymore and we can just trust what we are given through the NodeClaim now

Reply:
Yes, this is a old one. We should remove this since we have changed how the results from the scheduler is handled for the minimum requirement.